psibomana/author-quiz
# Component setup with static values

What does this PR do?
Setup the Author Quiz Components with static values

Description of Task to be completed?
The author quiz should show on every refresh an author image and a list of titles to chose from

How should this be manually tested?
- Clone and set up the project using the instruction in the README.
- Use $ git checkout author-quiz-component-setup to check out the branch containing the changes of this pull request
- Use $ yarn start to start the application
- Visit http://localhost:3000/ to see the updated react application page.
